src/libraries/Common/tests/System/Net/EnterpriseTests/setup/apacheweb/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-20220421022739-9c434db
```

Comment:
Is there no mcr version of ubuntu with the floating tag? Or is there a better image to use that rolls in any of the vetted registries?

Reply:
no, there isn't a floating version on mcr. we're already using this image in a couple other places in the build.